haskell/src/CLI.hs

```diff
@@ -84,15 +88,61 @@ getURL url =
       url' = if hasScheme then url else "http://" <> url
    in toText url'
 
+mkSubCommand :: String -> String -> Parser a -> O.Mod CommandFields a
+mkSubCommand name doc parser = command name $ info (parser <**> helper) $ progDesc doc
+
+---------------------------------------------------------------
+-- Janitor cli
+---------------------------------------------------------------
+usageJanitor :: Parser (IO ())
+usageJanitor =
+  subparser
+    ( mkSubCommand "update-idents" "Update author identities" janitorUpdateIdent
+    )
+  where
+    janitorUpdateIdent = io <$> parser
+      where
+        parser = (,,) <$> configOption <*> elasticOption <*> workspaceOption
+        io (configPath, elasticUrl, workspaceNameM) = do
+          setEnv "CRAWLERS_API_KEY" "nokey" -- set a fake env var to resolv the configuration
```

Comment:
arg that is annoying, perhaps we need  config loader option to disable the environment lookup...